def log_error(error):
    print(error)  # Todo: log error
